rakefile

```diff
@@ -4,6 +4,9 @@ task :default => :performance
 
 desc 'Run Performance setup'
 rototiller_task :performance do |t|
+  raise "The BEAKER_INSTALL_TYPE env var must be set to either 'pe' or 'foss' to continue" unless
+      ENV['BEAKER_INSTALL_TYPE'] == 'pe' || ENV['BEAKER_INSTALL_TYPE' == 'foss']
+
   t.add_env({:name => 'BEAKER_PE_DIR', :message => 'The PE download directory, example: http://enterprise.delivery.puppetlabs.net/archives/releases/2017.2.4'}) if ENV['BEAKER_INSTALL_TYPE'] == 'pe'
```

Comment:
i just noticed we don't add_env BEAKER_INSTALL_TYPE to rototiller's env vars.  is that intended?

Reply:
Its down on line 104, it's a cmd line option.

Reply:
oops.  👍